lib/error-helper.js

```diff
@@ -18,6 +23,27 @@ const checkErrorResponse = function(badgeData, err, res, notFoundMessage = 'not
   }
 };
 
+checkErrorResponse.asPromise = function ({ notFoundMessage } = {}) {
+  return async function ({ buffer, res }) {
```

Comment:
Does this have to be async? It's not doing anything asynchronously.

Reply:
I think if it weren't async, it would not return a promise.